careers/ladder.md

```diff
@@ -28,73 +28,161 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
                   M1  > M2  > M3  > M4 > E1 > E2
 ```
 
+### Evaluation criteria
+
+- Knowledge Leadership
+  - Level of knowledge in a specific area
+  - Breadth and depth of technical and soft skills
+- Impact
+  - Nature and impact of problems solved and decisions made
+  - Level of judgment, analysis or creativity required
+- Influence
+  - Nature and level of influence on projects, strategy, and internal and external contacts
+- Discretion
+  - Nature of guidance received
+  - Nature of guidance provided to others
+
 ## Individual Contributors
 
 Individual contributor engineers define a "maker" role at Artsy where a person is responsible for writing code on a
 regular basis and has no direct reports. As an individual contributor engineer grows at Artsy the scale and impact
 of the code they write and systems they maintain is expected to increase.
 
+From one level to another, the scope of direct impact gradually increases and so does the sphere of influence. Increased
+mastery is also a necessary condition but it's not sufficient to expend impact.
+```
+Scope of impact:   Self   >   Team   > Teams > Org
+Level          : IC0, IC1 > IC2, IC3 >  IC4  > IC5
+```
+
 ### IC0 Intern
 
+#### tl;dr: focuses on learning core skills and applying them to deliver business value
+
+#### Knowledge Leadership
 - Has some knowledge of CS concepts.
-- Learns to complete well-defined and subdivided tasks.
 - Focuses on learning tools and processes.
 - Pushes code through the entire lifecycle from development to production.
+
+#### Impact
+- Learns to complete well-defined and subdivided tasks.
+
+#### Influence
+
 - Develops individual contributor skills, including source control, editors or test-driven development.
+- Seeks assistance and feedback on a regular basis with teammates and manager
+
+#### Discretion
+- Receives active guidance from peers and manager.
 
 ### IC1 Associate Engineer
 
+#### tl;dr: focuses on being an effective individual contributor
+
+#### Knowledge Leadership
 - Has broad knowledge of CS concepts.
-- Can complete well-defined and subdivided tasks.
 - Focuses on learning tools and processes.
+
+#### Impact
+- Can complete well-defined and subdivided tasks.
+
+#### Influence
 - Routinely pushes code through the entire lifecycle from development to production.
 - Continues to develop individual contributor skills, including source control, editors or test-driven development.
+- Understands team mission, goals and backlog.
+
+#### Discretion
+- Guided by objectives of project or ongoing assignments.
+- Plans own work priorities on a regular basis.
+
 
 ### IC2 Engineer
 
+#### tl;dr: is an effective team member
+
+#### Knowledge Leadership
 - Consistently writes and delivers correct and clean quality code with guidance.
+- Owns a small-to-medium feature from technical design through completion.
```

Comment:
would be nice to have examples of what could be a small or medium feature. Maybe if all teams do sizing of the user stories the same way, we could use that as rough sizing of these features?